docs/src/docs/welcome/install.mdx

```diff
@@ -164,6 +164,76 @@ These installations aren't recommended because of the following points:
 go install github.com/golangci/golangci-lint/v2/cmd/golangci-lint@{.LatestVersion}
 ```
 
+<details>
+<summary>`go tool` usage recommendations</summary>
+
+We don't recommend using `go tool`.
+
+But if you want to use `go tool` to install and run golangci-lint (once again we don't recommend that),
```

Comment:
This mentioned in the previous sentence, no need to repeat: 
```suggestion
But if you want to use `go tool` to install and run golangci-lint,
```

Reply:
The repetition is intentional.